CHANGELOG.md

```diff
@@ -184,6 +184,90 @@ Released on: 2023-02-10
 
 Tout changement important à ce projet sera consigné dans le présent fichier.
 
+## [1.5.0](https://github.com/cds-snc/gcds-utility/compare/gcds-utility-v1.4.0...gcds-utility-v1.5.0)
+
+Publié le :  2024-12-05
+
+### :rocket: Nouvelles fonctionnalités
+
+* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191)) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
+
+### :rotating_light: Changement non rétrocompatibles
+
+* mettre à jour le cadre utilitaire avec de nouvelles typographies + unités de style d'espacement ([#197](https://github.com/cds-snc/gcds-utility/issues/197)) ([f6c2a24](https://github.com/cds-snc/gcds-utility/commit/f6c2a248547be78cb800fbb5770de99e457d8e30))
+
+Dans le cadre de nos efforts continus pour s'aligner sur les spécifications de Canada.ca, nous avons mis à jour nos échelles d'espacement et de typographie. Ces mises à jour ont été introduites avec [la dernière version de nos unités de style de conception](https://github.com/cds-snc/gcds-tokens/blob/main/CHANGELOG.md#200).
+
+### Changement d’espacement
+
+* De nouvelles classes d'espacement sont maintenant disponibles
+* Les valeurs de toutes les classes précédentes ont changé
+  - `margin` (marge)
+  - `padding` (remplissage)
```

Comment:
```suggestion
  - `margin` (marge)
  - `padding` (remplissage)
```

I added the french in parenthesis, originally it was:
```suggestion
  - marge
  - remplissage
```

Reply:
Yes but can you swap out "remplissage" for "marges intérieures"? The translation team uses "marge intérieure" for "padding".

Reply:
```suggestion
  - `margin` (marge)
  - `padding` (marges intérieures)
```